packages/dd-trace/test/remote_config/index.spec.js

```diff
@@ -0,0 +1,608 @@
+'use strict'
```

Comment:
Any way to write these tests as integration tests and use the tracer directly instead of adding more unit tests?

Reply:
As we discussed privately, I'm totally up for it but I think we have to initiate this push with examples and tests we can copy from, then sync to all start the effort together.